packages/modules/electricity_tariffs/group_e/tariff.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
+from datetime import datetime, timezone, timedelta
+from dateutil import tz
+from urllib.parse import quote
+from typing import Dict
+from modules.common import req
+from modules.common.abstract_device import DeviceDescriptor
+from modules.common.component_state import TariffState
+from modules.electricity_tariffs.groupe_e.config import GroupeETariffConfiguration
+from modules.electricity_tariffs.groupe_e.config import GroupeETariff
```

Comment:
```suggestion
from modules.electricity_tariffs.group_e.config import GroupeETariffConfiguration
from modules.electricity_tariffs.group_e.config import GroupeETariff
```
Typo, dann läuft auch der Test durch.